BUILD.gn

```
@@ -435,6 +447,9 @@ if (current_toolchain != "${dir_pw_toolchain}/default:default") {
     if (enable_linux_tv_app_build) {
       deps += [ ":linux_tv_app" ]
     }
+    if (enable_linux_tv_casting_app_build) {
+      deps += [ ":linux_tv_app" ]
```

Not sure what this line does but should this be the following:

```suggestion
      deps += [ ":linux_tv_casting_app" ]
```

Wondering if this was a copy/paste error...

---

Ah.. yes. Thanks for catching that.